test/04_openst/gnosisSafeModules/delayedRecoveryModule/resetRecoveryOwner.js

```diff
@@ -9,17 +9,24 @@ const OpenST = require('../../../../src/openst');
 
 describe('DelayedRecoveryModule', async () => {
   it('resets recovery owner', async () => {
-    const chainConfig = shared.chainConfig;
-    const connection = shared.connection;
+    const { chainConfig } = shared;
+    const { connection } = shared;
     const web3 = connection.auxiliaryWeb3;
 
     // Required but not validated arguments
     const mockAddresses = {
-      organization: "0x0000000000000000000000000000000000000001",
-      utilityToken: "0x0000000000000000000000000000000000000001",
-      owner: "0x0000000000000000000000000000000000000002",  // 0x1 reserved for sentinels by Gnosis Safe
-      newRecoveryOwner: "0x0000000000000000000000000000000000000001",
-    }
+      organization: '0x0000000000000000000000000000000000000001',
+      utilityToken: '0x0000000000000000000000000000000000000001',
+      owner: '0x0000000000000000000000000000000000000002', // 0x1 reserved for sentinels by Gnosis Safe
+      newRecoveryOwner: '0x0000000000000000000000000000000000000001',
+    };
+
+    chainConfig.update({
+      utilityBrandedTokens: [{
+        address: mockAddresses.utilityToken,
+        organizationAddress: mockAddresses.organization,
+      }],
+    });
```

Comment:
I suggest we should not make above change. Instead we should create ticket to update this smoke test to test with real deployed addresses and not mock addresses. 
Already setup of openst is being done with "setup openst mock test". We can reuse the deployed addresses from chainConfig.
@jasonklein what do you think?